api/app/Models/PoolCandidate.php

```diff
@@ -21,6 +21,8 @@
  * @property boolean $is_indigenous
  * @property boolean $is_visible_minority
  * @property boolean $has_diploma
+ * @property object $equityObject
```

Comment:
Neither of these are actually properties of the poolCandidate model and should probably be removed.